multicluster/controllers/multicluster/member/serviceexport_controller.go

```diff
@@ -199,6 +200,16 @@ func (r *ServiceExportReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		return ctrl.Result{}, err
 	}
 
+	// The ExternalName type of Service is not supported since it has no ClusterIP
+	// assgined to the Service.
+	if svc.Spec.Type == corev1.ServiceTypeExternalName {
```

Comment:
I remember we check ClusterIP is valid or not somewhere. No? Should we check that instead?

Reply:
We had some codes to check available Endpoints, no check for ClusterIP.
I think we need both, I added a few lines to check Service with empty IP.